### ISSUE TITLE
Chatterino is moving the mouse, preventing regular WM operations

### ISSUE THREAD

**Issue by hollunder** [Chatterino/chatterino2#3837]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

I have a two monitor setup. When chatterino is running, it prevents me from switching window focus between monitors. Or rather, it immediately reverts what I do.
1. Have chatterino open on the right screen.
2. Focus a window on the left screen.
3. Move the mouse onto the chatterino window
4. Chatterino somehow moves the mouse back to the focused window.

I would argue that a program should never override window manager or regular mouse operations, but that is exactly what is happening here.

### OS and Chatterino Version

Arch Linux, Sway, chatterino2-git-r3578.881986d86-1

**Comment by pajlada** [comment]
We don't make any mouse movements, the only thing we do that I can imagine breaks this is focus requesting through the [QWidget::setFocus](https://doc.qt.io/qt-6/qwidget.html#setFocus) function.

I don't know off the top of my head how to debug this issue, happy to help answer any questions anyone else might have

**Comment by hollunder** [comment]
I could narrow it down a bit. There is a flash on the left screen, which may indicate that chatterino tries to show a popup/window there.
Also it happens when I move the mouse over specific parts of chatterino. For example it mostly does not happen in chat, except if I move the mouse over some icon (e.g. mod icon). It happens reliably if I move the cursor over the title line of a tab (not the tab itself, but the thing below).
So basically, whenever chatterino tries to show an overlay pop-up.
So I think this might be a duplicate of https://github.com/Chatterino/chatterino2/issues/3501.

**Comment by pajlada** [comment]
If it happens when you move your mouse over something like an emote or badge then yeah I suspect it's related to #3501 too

**Comment by pajlada** [comment]
Duplicate of #3501
